semgrep-core/Analyzing/Dataflow_constness.ml

```diff
@@ -362,14 +362,5 @@ let update_constness (flow :F.cfg) mapping =
            | Some c -> refine_constness_ref constness c
           )
       | ___else___ -> ()
-    );
-
-    (* Update LHS constness according to the output env. *)
```

Comment:
maybe you can keep a comment saying "we used to update the LHS constness according to the output env
but in x = E, x is a label, and should not be substituted for its value".